process_jartic_simple.py

```
@@ -0,0 +1,163 @@
+            # Update description
+            pbar.set_description(f"Processing {archive.name}")
+            
+            # Create a sub-progress tracking function
+            def update_progress():
+                nonlocal total_measurements
+                total_measurements += 1
+                pbar.update(1)
+                
+                # Update file size every 10k measurements
+                if total_measurements % 10000 == 0:
+                    if output_path.exists():
```

Ruff (0.12.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)